runtime/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/handler/SchemaRegistryCatalogHandler.java

```diff
@@ -97,7 +98,7 @@ public SchemaRegistryCatalogHandler(
         VaultHandler vault = supplyVault.apply(catalog.vaultId);
 
         HttpClient client;
-        if (vault != null)
+        if (this.baseUrl.regionMatches(true, 0, HTTPS, 0, 5))
```

Comment:
```suggestion
        if (this.baseUrl.startsWith("https://"))
```

FYI - we don't tolerate case-insensitive https scheme anywhere else.